main.go

```diff
@@ -49,10 +51,23 @@ func mkconf() Config {
 		}
 	}
 
-	dbstring, exists := os.LookupEnv("GODRINK_DB")
-	if !exists {
-		log.Fatal("No database given, exiting!")
+	dbdriver, driverExists := os.LookupEnv("GODRINK_DBDRIVER")
+	dbstring, connExists := os.LookupEnv("GODRINK_DB")
+	if !driverExists {
+		log.Println("No database driver given, using embedded sqlite.")
+		dbdriver = "sqlite3"
+		dbstring = "file::memory:?cache=shared"
```

Comment:
If I understand this correctly, this means using SQLite in-memory without a file?

While fine for tests, this looks (albeit only slightly) dangerous for unsuspecting users.
If they just start the thing, they'll get SQLite (which is fine) but they'll also lose their data once the process stops (which might, but might also not be fine!)

I'd prefer using (creating it if it doesn't exist) an actual file as a default, with an option (and corresponding warning in the terminal) to change to in-memory.

Reply:
Personally, I like the idea that with no explicit configuration, no clutter is created on disk. To reduce the risk of a data loss due to human error, what do you think about just changing the warning to explicitly state that no data is persisted in case an in-memory database is used?

Reply:
I'd be fine with just a strong warning if you prefer.

I just slightly dislike the thought of having no recourse after having accidentally started the thing with the wrong config. I admit, the chance of that ever becoming a true problem is very slim, but I lean towards "safe" defaults.

Reply:
Since we automatically create tables on application startup, and might perform  database migrations in the future, the safest option could actually be to not touch any files by default.